tutorials/plot_bem.py

```diff
@@ -23,23 +25,31 @@
     )
 
 ########################################################
-# First, create the required anatomical files
-
-hcp.make_mne_anatomy(**anatomy_params)
+# This function assumes that we have created the required anatomical files.
+# A simple way for doing that is using the function hcp.make_mne_anatomy;
+# it takes similar parameters as the hcp.compute_forward_stack function
+# we will be using next, so you could just run hcp.make_mne_anatomy(**anatomy_params)
+# to set up the anatomy files.
 
-########################################################
-# Next, compute the 'forward stack':
+# But for now, we first, compute the 'forward stack':
 # With one handy function, construct everything required for inverse modelling.
 # Results are stored in a dict.
 # Note this step can take up to a few minutes;
-# particularly calculating the forward model is computationally expensive.
+# particularly calculating the forward model/morphing is computationally expensive.
 
 forward_results = hcp.compute_forward_stack(**anatomy_params)
 print(forward_results.keys())
```

Comment:
The thing is, jona, that we don't need to compute the forward to get the bem. In order to giver visitors/users/devs a feeling how things work I'd prefer to pull out the three lines to compute the bem from that function. This would give us some further room to think what to do with this function and how to name it.